R/g.part2.R

```diff
@@ -44,7 +44,7 @@ g.part2 = function(datadir = c(), metadatadir = c(), f0 = c(), f1 = c(),
       params_247[["qwindow"]] = qwindow
     }
     rm(qwindow)
-
+    
```

Comment:
please remove all new empty lines in this and other functions of this PR.
I generally try to avoid having many empty lines in functions. It only requires more scrolling when navigating functions.